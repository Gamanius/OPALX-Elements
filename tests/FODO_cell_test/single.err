Error> 
Error> *** User error detected by function "OrbitThreader::checkElementLengths"
Error>     The time step is too long compared to the length of the
Error>     element 'FOCUS_POLE_1'
Error>     The length of the element is: 0.010000
Error>     The distance the particles drift in 10 time step(s) is: 0.028213
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
with errorcode -100.

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** STEP 24905676.0 ON merlin-c-010 CANCELLED AT 2025-06-19T17:42:55 ***
slurmstepd: error: *** STEP 24905676.0 ON merlin-c-010 CANCELLED AT 2025-06-19T17:42:55 ***
srun: error: merlin-c-010: task 0: Killed
