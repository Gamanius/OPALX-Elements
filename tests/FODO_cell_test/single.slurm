#!/bin/bash
#SBATCH --partition=hourly      # Using 'hourly' will grant higher priority
#SBATCH --nodes=1               # No. of nodes
#SBATCH --ntasks-per-node=1     # No. of MPI ranks per node. Merlin CPU nodes have 44 cores
#SBATCH --cpus-per-task=22      # No. of OMP threads
#SBATCH --time=00:45:00         # Define max time job will run (e.g. here 5 mins)
#SBATCH --hint=nomultithread    # Without hyperthreading
##SBATCH --exclusive            # The allocations will be exclusive if turned on (remove extra hashtag to turn on)

#SBATCH --output=single.out  # Name of output file
#SBATCH --error=single.err    # Name of error file

export OMP_NUM_THREADS=22
export OMP_PROC_BIND=spread
export OMP_PLACES=threads


srun --cpus-per-task=22  /data/user/binder_j/opalx-elements/build/src/opalx opalx/multi.in --info 10
