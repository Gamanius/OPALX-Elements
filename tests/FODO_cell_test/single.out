Warning: Option 'opalx/multi.in' is not parsed by Ippl.
OPAL-X> configure done argc= 4
OPAL-X>               ____  _____        _       __   __
OPAL-X> 
OPAL-X> This is OPAL-X (Object Oriented Parallel Accelerator Library) Version 2024.1.00
OPAL-X>             git rev. 01389cb44db5c13da929eb3a3ae955738cb80a98
OPAL-X> 
OPAL-X> 
OPAL-X>                      (c) PSI, http://amas.web.psi.ch
OPAL-X> 
OPAL-X> 
OPAL-X> This is the performance portable version of OPAL
OPAL-X> 
OPAL-X> Please send cookies, goodies or other motivations (wine and beer ... ) 
OPAL-X> to the OPAL developers opal@lists.psi.ch
OPAL-X> 
OPAL-X> Time: 17:42:55 date: 19/06/2025
OPAL-X> 
OPAL-X> * Host:   OpenMP
OPAL-X> * Device: OpenMP
OPAL-X> 
OPAL-X> * Reading input stream 'opalx/multi.in'
OPAL-X> 
OPAL-X> value: {EDES,P0,OPALVERSION} = {0.001, 0.00142197, 2.02401e+07}
OPAL-X> 
OPAL-X> * Number of distributions  1
OPAL-X> 
OPAL-X> * ************* D I S T R I B U T I O N ********************************************
OPAL-X> * 
OPAL-X> * Distribution type: GAUSS
OPAL-X> * 
OPAL-X> * SIGMAX     = 0.001 [m]
OPAL-X> * SIGMAY     = 0.001 [m]
OPAL-X> * SIGMAZ     = 0.001 [m]
OPAL-X> * SIGMAPX    = 0 [Beta Gamma]
OPAL-X> * SIGMAPY    = 0 [Beta Gamma]
OPAL-X> * SIGMAPZ    = 0 [Beta Gamma]
OPAL-X> * 
OPAL-X> * Distribution is injected.
OPAL-X> * 
OPAL-X> * **********************************************************************************
OPAL-X> 
OPAL-X> * ************* F I E L D S O L V E R ********************************************** 
OPAL-X> * FIELDSOLVER  FS1
OPAL-X> * TYPE         NONE
OPAL-X> * RANKS       1
OPAL-X> * NX           16
OPAL-X> * NY           16
OPAL-X> * NZ           16
OPAL-X> * BBOXINCR     1
OPAL-X> * GREENSF      INTEGRATED
OPAL-X> * XDIM         serial  
OPAL-X> * YDIM         serial  
OPAL-X> * ZDIM      parallel  
OPAL-X> * ********************************************************************************** 
OPAL-X> 
OPAL-X> * ************* B E A M ************************************************************ 
OPAL-X> * BEAM        BEAM1
OPAL-X> * PARTICLE    ELECTRON
OPAL-X> * REST MASS   0.000510999 [GeV]
OPAL-X> * CHARGE      -e * 1 
OPAL-X> * MOMENTUM    1.42197e+06 [eV/c]
OPAL-X> * MOMENTUM    0.00142197 [GeV/c]
OPAL-X> * CURRENT     1.3 [A]
OPAL-X> * FREQUENCY   1300 [MHz]
OPAL-X> * NPART       10000
OPAL-X> * ********************************************************************************** 
OPAL-X> 
OPAL-X> PartBunch Constructor
OPAL-X> * Solver and Load Balancer set
OPAL-X> number_of_processors 88
OPAL-X> MPI_Comm_size 1
OPAL-X> * About to create particles ...
OPAL-X> * Seed = 1234567 on all ranks
OPAL-X> * Particle creation done
OPAL-X> 
OPAL-X> * ************** B U N C H ********************************************************* 
OPAL-X> * PARTICLES       = 10000
OPAL-X> * CHARGE          = -1.000000e-09 (Cb) 
OPAL-X> * INTEGRATOR      = LF2
OPAL-X> * MIN R (origin)  = (     -3.01299 ,     -2.98830 ,     -2.99130 ) [mm]
OPAL-X> * MAX R (max ext) = (      2.97979 ,      2.97993 ,      2.98913 ) [mm]
OPAL-X> * RMS R           = (    986.19027 ,    970.73298 ,    987.34158 ) [um]
OPAL-X> * RMS P           = ( 0.000000e+00 , 0.000000e+00 , 1.865175e-14 ) [beta gamma]
OPAL-X> * Mean R: ( 4.553649e-21 , -1.578598e-20 , -1.925543e-20 ) [m]
OPAL-X> * Mean P: ( 0.000000e+00 , 0.000000e+00 , 2.782725e+00 ) [beta gamma]
OPAL-X> * MESH SPACING    = (    378.29393 ,    376.74496 ,    377.51456 ) [um]
OPAL-X> * COMPDOM INCR    = 1.000000e+00 (%) 
OPAL-X> * FIELD LAYOUT    = Domain = {[0:15:1],[0:15:1],[0:15:1]}
OPAL-X> Total number of boxes = 1
OPAL-X>     Box 0 {[0:15:1],[0:15:1],[0:15:1]}
OPAL-X> 
OPAL-X> * Centroid : 
OPAL-X> * 4.553649e-17 0.000000e+00 -1.578598e-16 0.000000e+00 -1.925543e-16 2.782725e+04 
OPAL-X> * Cov Matrix : 
OPAL-X> * 9.725712e-07 0.000000e+00 -1.058310e-09 0.000000e+00 -5.476132e-09 4.437343e-35 
OPAL-X> * 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 
OPAL-X> * -1.058310e-09 0.000000e+00 9.423225e-07 0.000000e+00 -6.373156e-09 2.908925e-34 
OPAL-X> * 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 
OPAL-X> * -5.476132e-09 0.000000e+00 -6.373156e-09 0.000000e+00 9.748434e-07 1.553070e-34 
OPAL-X> * 4.437343e-35 0.000000e+00 2.908925e-34 0.000000e+00 1.553070e-34 3.478877e-28 
OPAL-X> * * ********************************************************************************** 
OPAL-X> * Creating new file for data: 'opalx/multi.stat'
OPAL-X> * Creating new file for data: 'opalx/multi.lbal'
OPAL-X> * FOCUS_POLE_1 gone live
OPAL-X> * DRIFT_1 gone live
OPAL-X> * DEFOCUS_POLE_1 gone live
OPAL-X> * DRIFT_2 gone live
OPAL-X> * FOCUS_POLE_2 gone live
OPAL-X> * DRIFT_3 gone live
OPAL-X> * DEFOCUS_POLE_2 gone live
OPAL-X> * DRIFT_4 gone live
OPAL-X> * FOCUS_POLE_3 gone live
OPAL-X> * DRIFT_5 gone live
OPAL-X> * DEFOCUS_POLE_3 gone live
OPAL-X> * DRIFT_6 gone live
OPAL-X> ParallelTrack: momentum=  2.78273
OPAL-X> itsBunch_m->RefPartR_m= ( 4.55365e-21 , -1.5786e-20 , -1.92554e-20 )
OPAL-X> itsBunch_m->RefPartP_m= ( 0 , 0 , 2.78273 )
OPAL-X> OrbitThreader dt_m= 1e-11
OPAL-X> * Finished time step: 0 time: 0
